### PR TITLE
Quickstart rewrite

### LINKED ISSUE #69: Issue trying to follow quickstart guide
I was following the steps at the [quickstart guide](https://macrometa.com/docs/quickstart/) and when I was trying to run the `Remove Contact` and `Update Contact` queries, I got the following errors:

image

image

I was following the tutorial using Google Chrome Version 102.0.5005.61 (Official Build) (x86_64)

### PR DESCRIPTION
**Pull request by diana-macrometa**
Completely rewrote and tested the quickstart guide.

Reviewers: I need some examples for a few of the API calls. They currently give errors when entered normally, per issue #69 

Fixes: #69, #70

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -42,67 +50,108 @@ FOR persons IN [
...
+### Create a new record.
 
-**SaveContact**
+**Name:** saveContact
 ```sql
 INSERT {firstname:@firstName,
         lastname:@lastName,
         email:@email} 
 INTO addresses
 ```
```
Comment (diana-macrometa) [inline docs/quickstart.md:130]: Need example for this.

File: docs/quickstart.md [hunk 2]
```diff
@@ -42,67 +50,108 @@ FOR persons IN [
...
-```sql
-REMOVE @_key 
-IN addresses
-```
+### Update a record.
 
-**UpdateContact**
+If you run this query as-is, with no key, you will see an error message if no key is entered. This is necessary to create the API endpoint. API calls would provide a specific key.
+
+**Name:** updateContact
```
Comment (diana-macrometa) [inline docs/quickstart.md:133]: Need example for this.

File: docs/quickstart.md [hunk 3]
```diff
@@ -42,67 +50,108 @@ FOR persons IN [
...
                     email:@email} 
 IN addresses
 ```
 
-Sweet, so now you have a full functional API for your app. [We made a front-end for you to take your new backend for a spin](https://github.com/Macrometacorp/tutorial-addressbook-restql).
+### Delete a record.
+
+If you run this query as-is, with no key, you will see an error message if no key is entered. This is necessary to create the API endpoint. API calls would provide a specific key.
+
+**Name:** removeContact
```
Comment (diana-macrometa) [inline docs/quickstart.md:143]: Need example for this.

**Review by elof — COMMENTED** [review]
On step 8 'Finish the api' we use bind parameters quite heavily. These are very important for making the API endpoints dynamic and we don't explain what they are, I think it would be useful to write a blurb about them and link to the docs: https://macrometa.com/docs/c8ql/fundamentals#bind-parameters

**Comment by diana-macrometa** [comment]
This PR is ready for final review.

**Review by jamesm-macrometa — APPROVED** [review]
Made some minor revisions to the Quickstart.  Looks good to me!

**Review by astrotars — APPROVED** [review]
👍

**Pull request merged**